rustfmt.toml

```diff
@@ -1,6 +1,13 @@
 version = "Two"
 unstable_features = true
 
+comment_width = 100
```

Comment:
@BusyJay @tabokie

Reply:
maybe we can set it to 120 as the resolution of screen is large enough nowadays? `100` width is not handy to write in a single line sometimes.

Reply:
for reference, risingwave use a width of 120 https://github.com/singularity-data/risingwave/blob/main/rustfmt.toml#L1

Reply:
What's the good of fitting it in one line? IMO the point is not the screen but the human reading experience.

E.g. I take a look at WSJ, NYT, Bloomberg. Their websites use line width ranging from 70 to 85. So I'll vote for 80 default here.

Reply:
Consider the comment for code in a deep nest, the indentation may already occupy 30 width. Then the space is too narrow to write comments. Though I'm not insisting on writing in one line. 

I don't vote for 80, as the code width is 100. we'd make the comment width at least the same as code width.

Reply:
I think different width makes sense in two ways:
- It looks good. Down to the bottom it's because we read code in structured tokens, but read text as unstructured words. So we have a higher tolerance for code width, but lower for text.
- Long-form comments always fill the width, whereas code rarely hits the limit. When setting the comment width the same as code, you will get bulky comments that visually stand out.

Reply:
The second point hits me, okay let's stick to rust default comment width. What do you think @BusyJay

Reply:
Note 80 doesn't count indentation, but 100 counts. So when using 80/100, it means:
1. The total length is within 100.
2. The length of comment itself is within 80.

I suggest to keep the default settings as most projects do.

Reply:
where says `The length of comment itself is within 80.`

Reply:
> Source lines which are entirely a comment should be limited to 80 characters in length (including comment sigils, but excluding indentation)

https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md

Reply:
Oh, it gives a quite reasonable answer to why the default comment width is not the same as the max line width